tests/testEdges.py

```diff
@@ -1,27 +1,48 @@
 #!/usr/bin/env python
-doPlot = False
-if doPlot:
-    import matplotlib
-    matplotlib.use('Agg')
-    import pylab as plt
-    import os.path
-    plotpat = os.path.join(os.path.dirname(__file__), 'edge%i.png')
-    print 'Writing plots to', plotpat
-else:
-    print '"doPlot" not set -- not making plots.  To enable plots, edit', __file__
-
+#
+# LSST Data Management System
+#
+# Copyright 2008-2016  AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
 import unittest
-import lsst.utils.tests         as utilsTests
 
 import numpy as np
 
+import lsst.utils.tests
 import lsst.afw.detection as afwDet
 import lsst.afw.geom as afwGeom
 import lsst.afw.image as afwImage
 import lsst.pex.logging as pexLogging
+import lsst.meas.algorithms as measAlg
 from lsst.meas.deblender.baseline import deblend
 
-import lsst.meas.algorithms as measAlg
+doPlot = False
+if doPlot:
+    import matplotlib
+    matplotlib.use('Agg')
+    import pylab as plt
+    import os.path
+    plotpat = os.path.join(os.path.dirname(__file__), 'edge%i.png')
+    print 'Writing plots to', plotpat
```

Comment:
It's not entirely clear how to handle these lines (and the similar lines in the other file). I guess we leave it as is for now until we can work out how `doPlot` ever becomes true when we are running in `py.test`. We could put this code in a `setupClass` method and trigger with an environment variable but ignore it for now. Please file a ticket for revisiting the plotting in the test code in `meas_deblender`. I think that's what @jdswinbank would prefer.